acms-react-app/src/Pages/Admin_Pages/Customers/CustomerInvoices_Page/CustomerInvoices.js

```diff
@@ -80,9 +73,6 @@ function CustomerInvoices() {
                         <input className="w-full rounded border-gray-300 px-4 py-2 focus:outline-none focus:border-indigo-500" type="text" placeholder="Search..." />
                         <span className="text-gray-400 cursor-pointer">&#128269;</span>
                       </div>
-                      <button className="text-white border-none px-4 py-2 rounded font-bold transition duration-300 hover:scale-110 bg-black">
-                      Add+
```

Comment:
If you deleted the add button, how does a user get to add a new invoice?

Reply:
I believe that for the context of invoice keeping the client wanted it so that creating, updating and deleting invoices are a functionality that should be exclusive to an admin account and that Customers should just be able to view all their past invoices in their account tab. We could have a discussion about maybe adding the functionality of creating an invoice from that flow as well in next scrum meeting